frontend/components/panels/WithdrawDAIPanel.vue

```diff
@@ -19,7 +19,7 @@
             <BaseButton :disabled="isWithdrawing" @click="$emit('manageVat')"> Manage DAI in VAT </BaseButton>
             <BaseButton
                 type="primary"
-                :disabled="!isWalletConnected || !isWalletAuthorized || !hasDaiToWithdraw || isRefreshing"
```

Comment:
Why did we remove the isRefreshing? If we are retrieving new values from the chain we don't want the user to execute any transactions as the data may be outdated. 

We also don't want the button to be loading while refreshing, as we use the loading indicator to show if the user is withdrawing.